Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
Yeah the reason why I made these changes was because I was getting the spaces around parameters reported as errors.

I'm on the latest dev build. I'll double check the version when I get back to my computer.